app/controllers/items/detail.js

```diff
@@ -386,6 +386,28 @@ export default GoodcityController.extend(
       }, ERROR_STRATEGIES.MODAL);
     },
 
+    itemValueHkDollar: Ember.computed(
+      "item",
+      "valueHkDollar",
+      "item.isBoxOrPallet",
+      function() {
+        if (this.get("item.isBoxOrPallet")) {
+          return false;
+        }
+
+        return this.get("valueHkDollar");
+      }
+    ),
+
+    isInvalidValuation: Ember.computed("itemValueHkDollar", function() {
+      const value = this.get("itemValueHkDollar");
+      return value === "" || value === null;
```

Comment:
```suggestion
      return Boolean(this.get("itemValueHkDollar"))
```

Reply:
Actually, it does not accept zero value also when we use `Boolean()`.